op-challenger/game/fault/trace/cannon/provider.go

```diff
@@ -123,7 +123,7 @@ func (p *CannonTraceProvider) GetStepData(ctx context.Context, pos types.Positio
 	}
 	var oracleData *types.PreimageOracleData
 	if len(proof.OracleKey) > 0 {
-		oracleData = types.NewPreimageOracleData(proof.OracleKey, proof.OracleValue, proof.OracleOffset)
+		oracleData = types.NewPreimageOracleData(0, proof.OracleKey, proof.OracleValue, proof.OracleOffset)
```

Comment:
This is probably a good place to use a TODO with issue number since we need to be sure to correct it once we know what the right value is.